packages/turbo/ios/RNVisitableView.swift

```diff
@@ -187,5 +192,14 @@ extension RNVisitableView: RNVisitableViewControllerDelegate {
     ]
     onLoad?(event)
   }
+    
+  func showVisitableActivityIndicator() {
+    guard !isRefreshing else { return }
```

Comment:
Do we need this guard clause?

Reply:
Otherwise we end up displaying loadingComponent when we refresh the webpage. [turbo-ios](https://github.com/hotwired/turbo-ios/blob/c9d060fcdec136c3808cd0764e956d375012c6fb/Source/Visitable/VisitableView.swift#L112-L117) does something similar.

Reply:
Allright, makes sense. I think the naming of the method just confuses me a bit then.